pages/[network]/proposal.tsx

```diff
@@ -63,6 +63,8 @@ export default function PageProposal() {
         }));
       })
       .catch(error => {
+        if (error?.code === 4001) return;
```

Comment:
what error is this? :o

Reply:
@moshmage is "rejected transaction", we don't need to show a toast if user rejects the transaction